plugin/geoip/geoip_test.go

```diff
@@ -0,0 +1,70 @@
+package geoip
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/coredns/coredns/plugin/test"
+	"github.com/coredns/coredns/request"
+	"github.com/coredns/coredns/plugin/metadata"
+)
+
+func TestGeoIPName(t *testing.T) {
```

Comment:
This doesn't really test anything useful, please remove

Reply:
Done